src/pages/Vault/index.tsx

```diff
@@ -70,12 +85,12 @@ const Vault = (): JSX.Element => {
     apy
   } = useSelector((state: StoreType) => state.vault);
   const [capacity, setCapacity] = React.useState(BitcoinAmount.zero);
-  const [feesEarnedPolkaBTC, setFeesEarnedPolkaBTC] = React.useState(BitcoinAmount.zero);
+  const [feesEarnedInterBTC, setFeesEarnedInterBTC] = React.useState(BitcoinAmount.zero);
```

Comment:
minor: missing a type here

Reply:
![Capture](https://user-images.githubusercontent.com/84005068/157736488-c3bf9092-2b3d-4e07-942a-620957468050.PNG)

This was as original and as we have a default value typed, I don't think we have to type it again. Implicitly it's typed.

Reply:
PolkaBTC was very old (before rebranding currency though) so we should get rid of the PolkaBTC names though. Thanks for pointing out @tomjeatt and thanks for fixing @crypto-engineer